doc/fwutil/fwutil.md

```diff
@@ -454,6 +486,24 @@ Warning: <firmware_update_notification>
 New FW will be installed, continue? [y/N]: N
 Aborted!
 ```
+3. Non modular chassis platform
+```bash
+root@sonic:~# fwutil --yes --image=next --boot=cold --immediate=yes
+Warning: <firmware_update_notification>
+...
+FW update in progress ...
+...
+root@sonic:~# fwutil --yes --image=next --boot=any --immediate=no
+Warning: <firmware_update_notification>
+...
+FW update in progress ...
+...
+root@sonic:~# fwutil --yes --image=next --boot=any --immediate=yes --component=ssd
```

Comment:
@sujinmkang What about modular platforms? AFAIK, there is no clear restrictions on component names, which means that different modules can share the same component name. And what should be the expected behaviour in that case?